module/tabbed.lua

```diff
@@ -8,9 +8,7 @@ In the function themselves, the same object is refered to as "tabobj" which is w
 you will often see something like: "local tabobj = some_client.bling_tabbed" at the beginning
 of a function.
 
---]]
-
-local awful = require("awful")
+--]] local awful = require("awful")
```

Comment:
That looks very weird but who am I to question the choices of your formatter.

Reply:
Yea, I just use default settings for my formatter